matrix-helpers.el

```diff
@@ -45,6 +45,24 @@ return any value."
                           ,(car slots))))
     (rec (nreverse slots))))
 
+(defmacro oset-multi (object &rest pairs)
+  "Set slot values for OBJECT.
+PAIRS should be of the form (SLOT VALUE SLOT VALUE...)."
+  (declare (indent defun))
+  `(progn
+     ,@(cl-loop for (slot value) on pairs by #'cddr
+                collect (list 'oset object slot value))))
+
+(defmacro a-get* (&rest keys)
```

Comment:
Are you using this anywhere, or is this just here for later use? If so, I think that's fine :)

Reply:
ah, this is used in the later PR, ignore me, :smile: